devito/compiler.py

```diff
@@ -452,7 +452,7 @@ def __init__(self, *args, **kwargs):
         self.cflags.remove('-std=c99')
         self.cflags.remove('-O3')
         self.cflags.remove('-Wall')
-        self.cflags += ['-fast', '-acc']
+        self.cflags += ['-std=c++11', '-fast', '-acc', '-mp']
```

Comment:
Out of interest, why the `-mp` flag?

Reply:
to compile OpenMP

Reply:
Oh, I thought that flag did something different sorry :p